Open training/src/Topic_0/Hospital_builder.java

```diff
@@ -0,0 +1,23 @@
+package Topic_0;
+
+public interface Hospital_builder {
+
+	public void buildFloors();
```

Comment:
All methods define in an interface are public. It is redundant to use the keyword `public`.